### PR TITLE
[libc++] Add base for LLVM 19 release notes

### PR DESCRIPTION
**Pull request by ldionne**
This will make it easier for folks who have patches that are not targeting LLVM 18 -- they can write the release notes in the LLVM 19 release notes immediately.

### CODE REVIEW

File: libcxx/docs/ReleaseNotes/19.rst [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+Deprecations and Removals
+-------------------------
+TODO
+
+
+Upcoming Deprecations and Removals
+----------------------------------
+
+LLVM 20
+~~~~~~~
```
Comment (mordante) [inline libcxx/docs/ReleaseNotes/19.rst:60]: Should we already copy this section of the LLVM 18 release notes.

**Review by mordante — COMMENTED** [review]
I didn't add this originally with the removal of 17. The documentation fails to build due to an referenced `.rst` file. Either we could wait until after branching or temporary rename the fix, maybe `.txt`.

File: libcxx/docs/ReleaseNotes/19.rst (hunk 1 above)
Comment (ldionne) [reply libcxx/docs/ReleaseNotes/19.rst:60]: My thinking was that we'd move them over as we actually deprecated them. But I'm open to whatever.

Comment (philnik777) [reply libcxx/docs/ReleaseNotes/19.rst:60]: I also think it's better to copy them now. That gives us a really nice visual of what we still want to remove in this release.

**Review by philnik777 — APPROVED** [review]

**Pull request merged**